wire-grpc-client/src/main/java/com/squareup/wire/GrpcClient.kt

```diff
@@ -128,7 +128,7 @@ class GrpcClient private constructor(
     // 2. write it to the stream (blocking)
     // 3. repeat. We also have to wait for all 2s to end before closing the writer
     CoroutineScope(Dispatchers.IO).launch {
-      val requestWriter = GrpcWriter.get(
+      val requestWriter = GrpcMessageSink.get(
```

Comment:
I was wondering why we create `GrpcMessageSource` using a constructor call and `GrpcMessageSink` using a factory method?

Reply:
Not any more!